discovery-infra/override_release_images.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import json
+import os
+
+from test_infra import utils, consts
+from semver import VersionInfo
+
+def main():
+    # Load default release images
+    with open(consts.OCP_RELEASE_IMAGES_PATH, 'r') as f:
+        release_images = json.load(f)
+
+    # Extract version from release image
+    release_image = os.getenv("OPENSHIFT_INSTALL_RELEASE_IMAGE")
+    ocp_full_version = utils.get_full_openshift_version_from_release(release_image)
+    ocp_semver = VersionInfo.parse(ocp_full_version)
+    ocp_version = "{}.{}".format(ocp_semver.major, ocp_semver.minor)
+
+    # Find relevant release image
+    release_image = [v for v in release_images
+        if v.get('openshift_version') == ocp_version and v.get('cpu_architecture') == "x86_64"][0]
+    release_image_index = release_images.index(release_image)
+
+    # Update release image
+    release_image["openshift_version"] = ocp_version
+    release_image["url"] = os.getenv("OPENSHIFT_INSTALL_RELEASE_IMAGE")
+    release_image["version"] = ocp_full_version
+    release_images[release_image_index] = release_image
+
+    # Save release images
+    with open(consts.OCP_RELEASE_IMAGES_PATH, 'w') as f:
+        f.write(json.dumps(release_images))
```

Comment:
I find it problematic that we need to change source code of assisted-service
For example, someone that overrides the first time, will have to remember to remove changes on the next run that doesn't require override 
Can't we have the value stored in OPENSHIFT_VERSIONS instead of changing the original values?

Reply:
Yeah, added the result to RELEASE_IMAGES env var instead.